package.json

```diff
@@ -46,5 +46,13 @@
     "svg-parser": "^2.0.4",
     "svgo": "^2.8.0",
     "typescript": "^4.8.3"
-  }
+  },
+  "prettier": {
+    "arrowParens": "avoid",
+    "bracketSpacing": true,
+    "printWidth": 80,
+    "semi": false,
+    "singleQuote": false,
+    "trailingComma": "es5"
+  },
```

Comment:
Trailing comma here makes this package invalid json. Remove to fix the build.

Reply:
done. by the way do we want "trailingComma": "es5" or es6?

Reply:
Let's just keep it consistent with Force for now.